target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-dlink-dap-2610.dts

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "D-Link DAP 2610";
+	compatible = "dlink,dap-2610";
+
+	soc {
+		edma@c080000 {
```

Comment:
Spacing is wrong here

Reply:
I don't understand how you want me to change it. Please explain.

Reply:
Sorry, it looks wrong here but looking at it in a proper editor revels that its normal.